source/openapi-admin-v3.yaml

```diff
@@ -199,6 +199,22 @@ paths:
                 properties:
                   access_token:
                     type: string
+        "401":
+          description: Invalid token
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - allOf: 
+                    - $ref: "#/components/schemas/Error"
+                    - properties:
+                        error:
+                          const: "invalid refresh token: incorrect token type"
+                      description: |-
+                        The request did not use bearer token
+                        authentication with a valid refresh token. Make
+                        sure the Authorization header contains the
+                        `refresh_token`, not the `access_token`.
```

Comment:
might wanna make this a component. i suspect other endpoints would have this error when the errors are fully documented.

Reply:
Agreed - thinking we can componentize as part of https://jira.mongodb.org/browse/DOCSP-26644